src/extension/content.ts

```diff
@@ -132,7 +132,7 @@ function render() {
 }
 
 function insertInDOM() {
-  const referenceEl = document.querySelector(".mr-source-target");
+  const referenceEl = document.querySelector(".detail-page-description");
```

Comment:
add all classnames in a contant file.

ex: in the contant file, this class can be written as ->

DETAIL_PAGE_DESCRIPTION = '.detail-page-description'